src/VisualStudio/Razor/project.json

```diff
@@ -0,0 +1,7 @@
+﻿{
+    "dependencies": {
+    },
+    "frameworks": {
+        "net46": { }
```

Comment:
So unless someone tells me otherwise I'm assuming that this correct and I should leave it as it.